scripts/benchmark/run.py

```diff
@@ -48,6 +50,24 @@
     bold_red,
 )
 
+NUM_BLOCKS_TO_BENCHMARK = 2
+NUM_TX_TO_BENCHMARK = 1
```

Comment:
Maybe use `2` transactions here, just to make sure that there are no crashes while looping over the transactions.

```suggestion
NUM_TX_TO_BENCHMARK = 2
```